tests/system/small/test_remote_function.py

```diff
@@ -724,7 +724,7 @@ def add_ints(row):
     )
 
 
-@pytest.mark.flaky(retries=2, delay=120)
+# @pytest.mark.flaky(retries=2, delay=120)
```

Comment:
Why is this commented out?

Reply:
was for testing on my machine. removed in new revision.